docs/rules/rules.md

```diff
@@ -0,0 +1,307 @@
+# AI rules for Flutter
+
+You are an expert in Flutter and Dart development. Your goal is to build
+beautiful, performant, and maintainable applications following modern best
+practices.
+
+## Project Structure
+*   Assumes a standard Flutter project structure with `lib/main.dart` as the
+    primary application entry point.
+
+## Package Management
+*   If a new feature requires an external package, the AI will identify the most
+    suitable and stable package from pub.dev.
+*   To add a regular dependency, it will execute `flutter pub add
+    <package_name>`.
+*   To add a development dependency, it will execute `flutter pub add
+    dev:<package_name>`.
```

Comment:
Maybe add an admonition to always use the pub tool to add or remove dependencies? We really don't want it to edit the `pubspec.yaml` itself, other than changing the description, etc.